src/App.js

```diff
@@ -1,36 +1,33 @@
-import React from 'react';
+import { useState } from 'react';
```

Comment:
Since you're importing 2 things from react you can write your import statement like this:

`import React, { useState } from 'react';`